docs/docs/loading-plugins-from-your-local-plugins-folder.md

```diff
@@ -2,25 +2,61 @@
 title: Loading Plugins from Your Local Plugins Folder
 ---
 
-Gatsby can also load plugins from your website's local plugins folder which is a folder named `plugins` in the website's root directory.
+Gatsby can load plugins from your website's local plugins folder, which is a folder named `plugins` in the website's root directory.
 
-```javascript:title=gatsby-config.js
-module.exports = {
-  plugins: [`gatsby-local-plugin`],
-}
+Consider this example Gatsby website's project structure, which includes a local plugin called `gatsby-local-plugin`:
+
+```
+/my-gatsby-site
+└── /src
+    └── /pages
+    └── /components
+<!-- highlight-start -->
+└── /plugins
+    └── /gatsby-local-plugin
+        └── /package.json
+        └── /gatsby-node.js
+<!-- highlight-end -->
+└── gatsby-config.js
+└── gatsby-node.js
+└── package.json
 ```
 
-If you want to reference a plugin that is not in the plugins folder then you could use something like the following:
+Like the name of the plugins folder implies, you can include multiple plugins in your local plugin folder.
+
+Including a local plugin in your plugins folder also requires a configuration step (just like a 3rd party plugin you've installed into your `node_modules` by running `npm install`); just as plugins installed from npm need to be included in your `gatsby-config`, you need to add the name of your local plugin to the plugins array as well:
 
 ```javascript:title=gatsby-config.js
 module.exports = {
   plugins: [
-    // Shortcut for adding plugins without options.
-    "gatsby-plugin-react-helmet",
-    {
-      // Standard plugin with options example
-      resolve: require.resolve(`/path/to/gatsby-local-plugin`),
-    },
+    `gatsby-third-party-plugin`,
+    `gatsby-local-plugin`, // highlight-line
   ],
 }
 ```
+
+## Verifying your plugin is loading
+
+To verify that your plugin is actually being found, you could add a small snippet of code like this to a `gatsby-node.js` file in the root of your plugin:
```

Comment:
```suggestion
To verify that your plugin is available for use in your Gatsby site, you can add a small snippet of code to a `gatsby-node.js` file in the root of your plugin:
```

Reply:
Is it worth mentioning that you may not have a gatsby-node file in this directory already? Trying to think of the order of operations/use case here.

Reply:
Good point, I think that would be a good aside to say you may need to add one.